README.md

```diff
@@ -34,13 +34,18 @@ The project currently has the following features:
 - Standard food tokens used for the trail
 - Standard ant colours used
 - Players can collect food from the anthill
-- Ants fill the anthill from top to bottom
+- Multiple options for anthill filling
+  * Top-down
+  * Bottom-up
+  * Specific order: 4, 2, 0, 3 then 1
+  * User's choice (Not featured in the original game)
+  * A random choice generator for the above options
```

Comment:
This last line doesn't match very well with the other options. The other options refer to the order of the filling of the anthill, whereas this line refers to who makes the choice of the other options, the players or random. I would put this information after `Multiple options for anthill filling`, e.g. `Multiple options for anthill filling. The players can select which option to use or ask for the game to randomly select one.`